modules/cnv-vm-rdp-console-web.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * TBD
+// cnv/cnv_users_guide/cnv-accessing-vm-consoles.adoc
 
 [id="cnv-vm-rdp-console-web_{context}"]
 = Connecting to the RDP console
```

Comment:
*_console.rdp_*  uses the 3.x filename highlighting. It should be enclosed in backticks. There are several instances in this file, so please fix them.